manager/tests/sim/main.rs

```diff
@@ -554,7 +554,8 @@ fn simulate_task_creation_agent_usage() {
     // look different.
     let mut root_runtime = root_account.borrow_runtime_mut();
     // Move forward proper amount until slot 1740
-    let block_production_result = root_runtime.produce_blocks(1780);
```

Comment:
This test seems to fail with the new value, but passes with 1780. Seems like this logic would be fine to remain the same even if timestamps are used